darwin/importer/formats/csvtagsvideo.py

```diff
@@ -0,0 +1,36 @@
+import csv
+from pathlib import Path
+from typing import List, Optional
+
+import darwin.datatypes as dt
+
+
+def parse_file(path: Path) -> Optional[List[dt.AnnotationFile]]:
+    if path.suffix != ".csv":
+        return
+
+    files = []
+
+    file_annotation_map = {}
+    with path.open() as f:
+        reader = csv.reader(f)
+        for row in reader:
+            try:
+                filename, tag, start_frame, end_frame = map(lambda s: s.strip(), row)
+            except ValueError:
+                continue
```

Comment:
In the future, we could log warnings and errors in a log file, which would be useful when debugging.
Right now, we can simply add a TODO comment since we're probably in a rush